public/assets/styles/palette.css

```diff
@@ -41,4 +41,6 @@
   /* https://www.systeme-de-design.gouv.fr */
   --systeme-design-etat-bleu: #000091;
   --systeme-design-etat-gris-survol: #f6f6f6;
+  --systeme-design-etat-gris-actif: #ededed;
+  --systeme-design-etat-contour-champs: #dddddd;
```

Comment:
Si tu mets #ddd cela devrait fonctionner aussi

Reply:
Je me suis dit que par souci d'homogénéité je préférais les 6 `d`.